scripts/buildkite/check-nixpkgs-fmt.sh

```diff
@@ -0,0 +1,8 @@
+
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+nixpkgs-fmt `git ls-files -- '*.nix'`
```

Comment:
```suggestion
nixpkgs-fmt `git ls-files -- '*.nix' ':!:nix/sources.nix'`
```
```nix/sources.nix``` is generated by `niv` which will print a warning if the files has changed (thinking it is not up-to-date).